docs/modules/ROOT/pages/contributor-guide/release-guide.adoc

```diff
@@ -69,6 +69,24 @@ $ mvn deploy -Papache-release -DskipTests -Denforcer.skip -Dquarkus.build.skip -
 * This will close the repository from future deployments and make it available for others to view
 * After that, a popup window is created, enter the name and version of the artifact being released in the `Description` field and finally click `Close`
 
```

Comment:
Do we already have a section updating the svn repo here: https://camel.apache.org/camel-quarkus/3.5.x/contributor-guide/release-guide.html#_update_the_apache_camel_quarkus_distribution_subversion_repository ?

Maybe it's about changing the order of release steps ?

Reply:
I don't think we want to be publishing stuff to `dist/releases` before the vote has passed. Hence I added a separate step to publish the staged artifacts to `dist/dev`.